Implement package changelogs

m-blaha opened the pull request.

m-blaha committed:
Implement Package.get_changelogs() method

jrohel commented on include/libdnf/rpm/package.hpp:
```diff
@@ -62,6 +62,17 @@ struct PackageId {
 
+struct Changelog {
+public:
+    explicit Changelog(time_t _timestamp, std::string _author, std::string _text)
```
Why are argument names prefixed with an underscore?

m-blaha replied:
Oh, thanks. I will change this before switching from draft to regular PR.

m-blaha committed:
dnfdaemon: Implement changelogs package attribute

m-blaha committed:
dnfdaemon: Use standard libdnf method to load repositories

- repository load flags are taken from session configuration
- use repo_sack.update_and_load_enabled_repos() for repository loading
